### PR TITLE
Improve json for new landing

### PR DESCRIPTION
**Pull request by Serabe**
This is needed to achieve the new design in the new landing.

Needs #98

### CODE REVIEW

File: layouts/json/single.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 {"Posts": [
   {{ range $k, $post := (sort $pages.Pages "Date" "desc") }}
     {{if $k}},{{end}}
-    {{ dict "title" $post.LinkTitle "description" $post.Description "link" $post.Permalink "date" ($post.Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML) | jsonify }}
+
+    {{ $author := index .Site.Params.authors .Params.author }}
+    {{ dict "author" $author.name "author_avatar" (absURL $author.thumbnail) "title" $post.LinkTitle "description" $post.Description "featured_image" (absURL $post.Params.image) "link" $post.Permalink "date" ($post.Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML) | jsonify }}
```
Comment (dpordomingo) [inline layouts/json/single.html]: Please, confirm me that you felt dirty with this oneliner, and that you hate Hugo as much than I do.

File: layouts/json/single.html [hunk 2]
```diff
@@ -2,6 +2,8 @@
 {"Posts": [
   {{ range $k, $post := (sort $pages.Pages "Date" "desc") }}
     {{if $k}},{{end}}
-    {{ dict "title" $post.LinkTitle "description" $post.Description "link" $post.Permalink "date" ($post.Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML) | jsonify }}
+
```
Comment (erizocosmico) [inline layouts/json/single.html]: leftover?

File: layouts/json/single.html (hunk 1 above)
Comment (erizocosmico) [inline layouts/json/single.html]: what the actual fuck

Comment (Serabe) [inline layouts/json/single.html]: I tried it in several lines and I got a segfault :(

File: layouts/json/single.html (hunk 2 above)
Comment (Serabe) [inline layouts/json/single.html]: I needed to add air to the code in some place. Otherwise, it would be a loop with three lines of WTF.

**Pull request merged**